.github/RELEASE_TEMPLATE.md

```diff
@@ -1,150 +1,89 @@
 
-<!-- A snapshot of the documentation at the time of releasing. -->
+3. (optional) Install the bash/zsh auto-completion script according to the [byron cli manual](https://github.com/input-output-hk/cardano-wallet/wiki/Wallet-Command-Line-Interface-(cardano-wallet-byron)/{{BYRON_CLI_WIKI_COMMIT}})
 
-| Link                                                                                                          | Audience                                                   |
-| ---                                                                                                           | ---                                                        |
-| [API Documentation](https://input-output-hk.github.io/cardano-wallet/api/vYYYY-MM-DD)                         | Users of the Cardano Wallet API                            |
-| [Haddock Documentation](https://input-output-hk.github.io/cardano-wallet/haddock/vYYYY-MM-DD)                 | Haskell Developers using the `cardano-wallet` as a library |
-| [CLI Manual](https://github.com/input-output-hk/cardano-wallet/wiki/Wallet-command-line-interface/<revision>) | Users of the Cardano Wallet API                            |
+4. Start `cardano-wallet --help` and see available parameters.
 
-# Changelog
+## Additional notes
 
-<!-- A CHANGELOG, organized in milestones. Ideally, we put it within
-some <details></details> elements to avoid cluttering the release notes -->
+- On macOS: Make sure all `*.dylib` files are in the same directory as `cardano-wallet` binary.
 
-# Bug Fixes
+### Docker
 
-<!-- Fixes included in this release that were present in the previous release.
-e.g.
+1. Pull from DockerHub.
 
-  - <bug description> [#NNNN](<link-to-ticket>)
--->
+```
+$ docker pull inputoutput/cardano-wallet:{{CABAL_VERSION}}-jormungandr
+```
 
-# Weekly Reports
+2. Verify the image using the command-line.
 
-<!-- A link to the relevant weekly report, giving extra details about
-development and progress -->
+```
+$ docker run --rm inputoutput/cardano-wallet:{{CABAL_VERSION}}-jormungandr version
+```
 
 # Signatures
 
 <!-- Signatures of people responsible for the release -->
 
 Name                           | Role                | Approval
 ---                            | ---                 | ---:
-Matthias Benkort @KtorZ        | Technical Team Lead | :hourglass:
-Piotr Stachyra @piotr-iohk     | QA Engineer         | :hourglass:
-Tatyana Valkevych @tatyanavych | Release Manager     | :hourglass:
+Matthias Benkort @KtorZ        | Technical Team Lead | :hourglass: 
+Piotr Stachyra @piotr-iohk     | QA Engineer         | :hourglass: 
+Tatyana Valkevych @tatyanavych | Release Manager     | :hourglass: 
 
-[new]: https://raw.githubusercontent.com/input-output-hk/cardano-wallet/master/.github/images/badge-new.png 
+[new]: https://raw.githubusercontent.com/input-output-hk/cardano-wallet/master/.github/images/badge-new.png
```

Comment:
I guess this badge is no longer useful since the "key features" section was dropped.